localstack/services/s3/s3_listener.py

```diff
@@ -508,6 +508,9 @@ def fix_range_content_type(bucket_name, path, headers, response):
     if 'Range' not in headers:
         return
 
+    if response.status_code == 404:
```

Comment:
nitpick: Should we return here for any type of error status code?
```
if response.status_code >= 400:
```

Reply:
Yes we should, but I don't have the full list. Should I add it for 4XX and 5XX?

Reply:
Yes, I think anything greater or equal to 400 should be fine (since that would indicate an error code..)